doc/source/cookbook/complex_plots.rst

```diff
@@ -88,7 +88,7 @@ Multipanel with Axes Labels
 This illustrates how to use a SlicePlot to control a multipanel plot.  This
 plot uses axes labels to illustrate the length scales in the plot.
 See :ref:`slice-plots` and the
-`Matplotlib AxesGrid Object <https://matplotlib.org/mpl_toolkits/axes_grid/api/axes_grid_api.html>`_
+`Matplotlib AxesGrid Object <https://matplotlib.org/stable/mpl_toolkits/axes_grid/api/axes_grid_api.html>`_
```

Comment:
Oh no 404!

Reply:
goof catch. I figured my one-liner could use more refinement but I'm now just fixing the 404 ones manually